jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/WebAppGenerator.java

```diff
@@ -232,4 +191,35 @@ private String getDeploymentDir(AppServerHandler handler) {
   private String getUser(AppServerHandler handler) {
     return getConfig(Config.USER, handler.getUser());
   }
+
```

Comment:
hello any reason why this part is moving at the end? I am not sure the formatter is enforcing it